modules/page_object_autofill_test_basic.py

```diff
@@ -1,8 +1,9 @@
-from modules.page_object_autofill import Autofill
+from modules.page_base import BasePage
 from modules.util import AutofillAddressBase, BrowserActions
+from selenium.webdriver.support import expected_conditions as EC
 
 
-class AddressFill(Autofill):
+class AddressFill(BasePage):
```

Comment:
My bad! Leave this as `class AddressFill(Autofill):` and change all the return types to `Autofill` instead of `BasePage`.